src/backend/updating-llvm.md

```diff
@@ -57,59 +52,72 @@ the branch we're already using. The steps for this are:
    src/llvm-project` typically.
 10. Wait for PR to be merged
 
-The tl;dr; is that we can cherry-pick bugfixes at any time and pull them back
-into the rust-lang/llvm-project branch that we're using, and getting it into the
-compiler is just updating the submodule via a PR!
+In summary,
+we can cherry-pick bugfixes at any time and pull them back
+into the rust-lang/llvm-project branch that we're using,
+and getting them into the compiler is just updating the submodule via a PR.
```

Comment:
I don't think this change makes any significant effect.

Reply:
removing...